custom_components/acwd/acwd_api.py

```
@@ -100,17 +131,25 @@ def login(self):
+                retry_delay=2.0,
+                json=login_payload,
+                timeout=30,
+                headers={
+                    'Content-Type': 'application/json; charset=UTF-8',
+                    'Referer': self.base_url,
+                    'X-Requested-With': 'XMLHttpRequest',
+                    'User-Agent': USER_AGENT,
+                    'CSRFToken': csrf_token
+                }
+            )
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)